docker-compose.yml

```diff
@@ -82,6 +82,7 @@ services:
     - ./docker/sphinx-start:/usr/local/bin/sphinx-start:ro
     - ./docker/sphinx-indexer:/usr/local/bin/sphinx-indexer:ro
     - sphinx_indexes:/opt/sphinx/indexes
+    profiles: [sphinx]
```

Comment:
The compose file isn't used for production deployments is it? Since in that case we'd need to adapt some stuff accordingly.